packaging/os/rudix.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, James Polley <jp@jamezpolley.com>
+# Based on macports (Jimmy Tang <jcftang@gmail.com), okpg (Patrick
+# Pelletier <pp.pelletier@gmail.com>), pacman (Afterburn) and pkgin
+# (Shaun Zinck) modules
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: rudix
+author: James Polley
+short_description: Package manager for Rudix
+description:
+    - Manages Rudix packages
+version_added: "1.9.2"
+options:
+    name:
+        description:
+            - name of package to install/remove
+        required: true
+    state:
+        description:
+            - state of the package
+        choices: [ 'present', 'absent', 'active', 'inactive' ]
+        required: false
+        default: present
+    update_cache:
```

Comment:
you removed the function, but not the docs nor the spec definition of the parameter